doc/data/messages/t/too-many-positional-arguments/details.rst

```diff
@@ -1,10 +1,11 @@
-Positional arguments work well for cases where the the use cases are
-self-evident, such as unittest's ``assertEqual(first, second, msg=None)``.
-Comprehensibility suffers beyond a handful of arguments, though, so for
-functions that take more inputs, require that additional arguments be
-passed by *keyword only* by preceding them with ``*``:
+Good function signatures don’t have many positional parameters. For almost all
```

Comment:
```suggestion
Good function signatures don’t have many positional parameters because, when the function is called with many arguments, it is difficult to understand what parameters they refer to.
```

Reply:
Applying this won't work with the line under it that starts with "interfaces, comprehensibility suffers beyond a handful of arguments.". Do you mind suggesting on the two lines instead ?

Reply:
Let's see if I can - Let me try hiding the comment first

Reply:
Oh I don't know how to suggest it again. No worries anyway we can iterate another time if we agree with it.